src/gateways/createBuyPrintConfigLoader.ts

```diff
@@ -18,6 +19,17 @@ export default (url: string) => {
         } else {
           return rejectResponse(response,  (status, message) => `Error response from BuyPrint ${status}: ${message}`);
         }
+      }).catch((e) => {
+        Sentry.captureException(e);
+
+        return Promise.resolve({
+          buy_urls: [{
+            allows_redirects: false,
+            disclosure: null,
+            provider: 'amazon',
```

Comment:
should we update this to something like `openstax fallback`?

Reply:
What exactly do you mean?

Reply:
the string `amazon` is not true here, we're directing folks to the openstax backend which will redirect them... somewhere. putting amazon might be confusing. something like `fallback` or `openstax` would be more clear

Reply:
Okay, i see:)

Reply:
the additional background here is that we send people to either amazon or shopify depending on their geographical location, so the provider is supposed to indicate which store they're going to.

in a normal situation the `buy_urls` would be direct urls to the provider, in this case since we're not getting the buy urls we'll send the user to the openstax backend, and it'll redirect them to whichever provider they're supposed to go to